patch_ng.py

```diff
@@ -1050,7 +1044,11 @@ def apply(self, strip=0, root=None):
 
         # check hunks in source file
         if lineno+1 < hunk.startsrc+len(hunkfind)-1:
-          if line.rstrip(b"\r\n") == hunkfind[hunklineno]:
+          raw_line = line.rstrip(b"\r\n")
+          patch_line = hunkfind[hunklineno]
+          if (raw_line == patch_line) or \
+             (fuzz and (not raw_line.startswith(b"+") and not patch_line.startswith(b"+")) and
+             (not raw_line.startswith(b"-") and not patch_line.startswith(b"-"))):
```

Comment:
What's the motivation for `raw_line.startswith(b"+")` or `raw_line.startswith(b"-")`? If `line` is the line from the original file, then I don't see any reason for those checks.

Reply:
nope, I've removed on latest patch. good eye! ;)